maxwellgithinji/Maintenance-Tracker
# #157787893 Implements admin's view of disapproved request

**What does this PR do?**
Enables the Admin to View the details of a disapproved user request

**Description of Tasks to be Completed**
_The Admin can:_
*  View the request title of a user.
* View User details
* View the feedback 
* View the request status
* View a Found Solution  button



**How Should This be Manually Tested?**
_After cloning the Repo:_
* Navigate to ```templates/admin_view_disapproved_detailed.html``` file and open it with a browser.
* Enter some details in the resolution comment field
* Enter some details in the user recommendation field
* Check the Resolve button
* Click on Found solution Button


**Any background context you want to provide?**
The responsiveness on small devices it yet  fully  implemented
There shouldn't be a submit button, but a Found solution button 

**See the Screenshots Below**
![bandicam 2018-05-27 09-57-37-318](https://user-images.githubusercontent.com/11288990/40583358-cac35036-6195-11e8-97b4-e4ec31bffe1b.jpg)
